tests/Integration/Behaviour/Features/Context/ProductFeatureContext.php

```diff
@@ -659,6 +659,9 @@ public function setProductCombinationMinimalQuantity(string $combinationName, st
     }
 
     /**
+     * This step was created when CQRS commands did not exist for product It shouldn't be used unless you want to force
+     * creating combinations in a "legacy way" and make sure they are consistent with the usage of new commands.
+     *
```

Comment:
Deprecated anotation here ?

Reply:
We can't really since it's still in use If not I would have completely removed it
I wanted to but I didn't want to modify the order tests for this And even if I did this step is still needed at least once because we have a behat test that assert "legacy products" (meaning products created with legacy code and not CQRS commands) behave as expected regarding their product type

This was needed to test use cases that can happen with upgraded shops